### ISSUE TITLE
Hard to repeat the training loss or accuracy as your trained model

### ISSUE THREAD

**Issue by wenston2006** [luoyetx/deep-landmark#11]
Hi, thanks for your sharing of the codes at first. I tried to use some new data (20,000 pics) to train the 1_F model through finetuning. I got the test output error of 0.006 during training, but I got a testing accuracy of 0.03-0.04% on five keypoints which are 0.01 more than your model. Moreover, as mentioned in other issues, I got different training loss during every new training period. So can you share some valuable experience on how to training the model ?  Can I try repeated finetuning? I mean using the model generated during last finetuning to do a new finetuing. Of course the newly trained model should have a lower training loss than the previous one.

**Comment by wenston2006** [comment]
I also found that the test output error during training would be kept at a value after 30,000 iterations, so I just do training for around 200,000 iterations. Should I keep doing training for 1000,000 iterations as the default setting in prototxt files?

**Comment by wenston2006** [comment]
Moreover ,can you send me one copy of your Face_alignment.pdf ?I failed to open this file.

**Comment by luoyetx** [comment]
The model I trained may use iters 1000,000 or 2000,000. The loss will still reduce when training goes on. Maybe you should plot the loss to see if it is.

for the pdf file, maybe something wrong with github contents, it should be the network error. I upload the pdf to [Baidu Driver](http://pan.baidu.com/s/1jIF8K0A).
